database/dummyOverview.js

```diff
@@ -0,0 +1,26 @@
+export default [
+  {
+    "title": "Mike Anderson",
+    "symbol": "💯",
+    "email": "mike.andersen@leu-rescue.org",
+    "phone": "970-631-6390",
+    "Date Joined": "none",
+    "2017 Status": "inactive",
+    "2018 Status": "none",
+    "Class": "Probationary",
+    "keywords":
+      "Mike Anderson mike.andersen@leu-rescue.org 970-631-6390 Date-Joined-null 2017-Status-inactive 2018-Status-none Class-Probationary"
```

Comment:
These keywords are simply for the search, correct?

Reply:
yes